app-i18n/fcitx/fcitx-5.1.5.ebuild

```diff
@@ -0,0 +1,120 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="fcitx5"
+
+inherit cmake xdg
+
+DESCRIPTION="Fcitx 5 is a generic input method framework"
+HOMEPAGE="https://fcitx-im.org/ https://github.com/fcitx/fcitx5"
+SRC_URI="https://download.fcitx-im.org/fcitx5/fcitx5/fcitx5-${PV}_dict.tar.xz -> ${P}.tar.xz"
+
+LICENSE="LGPL-2+ Unicode-DFS-2016"
+SLOT="5"
+KEYWORDS="~amd64 ~x86"
+IUSE="+autostart doc +emoji +enchant +keyboard presage +server systemd test wayland +X"
+REQUIRED_USE="
+	|| ( wayland X )
+	X? ( keyboard )
+	wayland? ( keyboard )
+"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	!app-i18n/fcitx:4
+	dev-libs/libevent
+	dev-libs/libfmt
+	doc? (
+		app-doc/doxygen
+		media-gfx/graphviz[svg]
+	)
+	emoji? ( sys-libs/zlib )
+	enchant? ( app-text/enchant:2 )
+	sys-apps/dbus
+	virtual/libintl
+	x11-libs/cairo[X?]
+	x11-libs/gdk-pixbuf:2
+	x11-libs/pango[X?]
+	keyboard? (
+		app-text/iso-codes
+		dev-libs/expat
+		dev-libs/json-c:=
+		x11-misc/xkeyboard-config
+		x11-libs/libxkbcommon[X?,wayland?]
+	)
+	systemd? ( sys-apps/systemd )
+	wayland? (
+		dev-libs/wayland
+		dev-libs/wayland-protocols
+		dev-util/wayland-scanner
+	)
+	X? (
+		>=x11-libs/xcb-imdkit-1.0.3:5
+		x11-libs/libX11
+		x11-libs/libXext
+		x11-libs/libXfixes
+		x11-libs/libXinerama
+		x11-libs/libXrender
```

Comment:
Is there any evidence to show `libXext`, `libXfixes`, `libXinerama` & `libXrender` packages are needed?

Reply:
I've deleted it. I also retested next in a clean environment.